packages/manager/apps/hycu/src/routes/routes.tsx

```diff
@@ -75,6 +75,17 @@ export const Routes: any = [
           },
         },
       },
+      {
+        id: 'order',
+        path: urls.order,
+        ...lazyRouteConfig(() => import('@/pages/order/Order.page')),
+        handle: {
+          tracking: {
+            pageName: 'order',
+            pageType: PageType.dashboard,
```

Comment:
tracking not correct ?

Reply:
Tracking will be implemented in another US